src/wizard/github.jl

```diff
@@ -5,20 +5,26 @@ using Registrator
 const _github_auth = Ref{GitHub.Authorization}()
 
 function github_auth(;allow_anonymous::Bool=true)
-    if !isassigned(_github_auth) || !allow_anonymous && isa(_github_auth[], GitHub.AnonymousAuth)
-        # If the user is feeding us a GITHUB_TOKEN token, use it!
-        if length(get(ENV, "GITHUB_TOKEN", "")) == 40
+
+    if (!isassigned(_github_auth) || !allow_anonymous && isa(_github_auth[], GitHub.AnonymousAuth)) && length(get(ENV, "GITHUB_TOKEN", "")) == 40
+        try 
             _github_auth[] = GitHub.authenticate(ENV["GITHUB_TOKEN"])
-        else
-            if allow_anonymous
-                _github_auth[] = GitHub.AnonymousAuth()
+        catch e
+            if occursin("401", e.msg)
+                @warn "GitHub was unable to authenticate using the token from ENV[\"GITHUB_TOKEN\"], it may be stale. We will now request a temporary token."
```

Comment:
Possibly anonymous auth may be attempted
```suggestion
                @warn "GitHub was unable to authenticate using the token from `ENV[\"GITHUB_TOKEN\"]`, it may be stale. Falling back on alternate authentication methods."
```